Hivemind/Assets/Scripts/SpatialPartitioning.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpatialPartitioning : MonoBehaviour
+{
+    public List<GameObject> Entities;
+    private List<SpatialPartitioning> Neighbors;
+
+    public int height;
+    public int width;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        Entities = new List<GameObject>();
+        Neighbors = new List<SpatialPartitioning>();
+        SetNeighbors();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        //TO DO, delete this when done DEBUGGING
+        if(Entities.Count != 0)
+        {
+            Debug.ClearDeveloperConsole();
+            Debug.Log(this.name+" Count: "+Entities.Count);
+        }
+    }
+
+    private void OnTriggerEnter(Collider entity)
+    {
+        Entities.Add(entity.gameObject);
+    }
+
+    private void OnTriggerExit(Collider entity)
+    {
+        Entities.Remove(entity.gameObject);
+    }
+
+    public List<GameObject> GetEntities()
+    {
+        return Entities;
```

Comment:
No need to get a property which is public, delete getter or make entities private